src/components/category-menu/index.css

```diff
@@ -38,6 +37,11 @@
 .category-menu-mobile {
     overflow-x: scroll;
     overflow-y: hidden;
+    height: 70.16px !important;
+}
+
+.category-menu-desktop{
+    justify-content: center !important;
```

Comment:
You can remove `!important` here. Not required.

**Always** try to avoid `!important` wherever possible.